Misc/NEWS.d/next/Library/2018-07-14-23-52-12.bpo-33668.7poHxA.rst

```diff
@@ -0,0 +1 @@
+Fix inspect.getmembers to travering __bases__ when the object is a class.
```

Comment:
`travering` -> `traversing`

And I would add "only" before "traversing".

And maybe add a markup: ``` :func:`inspect.getmembers` ```.